lib/validators/options_processor/teams.js

```diff
@@ -0,0 +1,47 @@
+const TeamNotFoundError = require('./../../errors/teamNotFoundError')
+const _ = require('lodash')
+
+class Teams {
+  static async extractTeamMembers (context, teams) {
+    let teamMembers = []
+    for (let team of teams) {
+      let members = []
+      try {
+        members = await getTeamMembers(context, team)
+      } catch (err) {
+        throw err
+      }
+
+      teamMembers = teamMembers.concat(members)
+    }
+    return _.uniq(teamMembers)
+  }
+}
+
+const getTeamMembers = async (context, team) => {
+  const stringArray = team.split('/')
+  if (stringArray.length !== 2) {
+    throw Error(`each team id needs to be given in 'org/team_slug'`)
+  }
+  if (stringArray[0].indexOf('@') === 0) stringArray[0] = stringArray[0].substring(1)
+
+  const org = stringArray[0]
+  const teamSlug = stringArray[1]
+
+  let res
+  try {
+    res = await context.github.teams.listMembersInOrg({
```

Comment:
What app permissions is required for this API call?

Reply:
`team:read` i've added it in docs